test/conformance/chainsaw/cli/rule-same-policy-different/policy.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: preconditions
+spec:
+  admission: true
+  background: false
+  rules:
+  - match:
+      any:
+      - resources:
+          kinds:
+          - Pod
+    name: anyrule
```

Comment:
This is very confusing when one rule has a dash and the other doesn't. Suggest making this more obvious.

Reply:
can we add comments ?

Reply:
Sure, but please use YAML style comments.